docs/index.mdx

```diff
@@ -46,6 +46,22 @@ Second, a GitPOAP is awarded - Contributors are awarded GitPOAPs (POAPs with an
 
 <HowSvg />
 
+### Building on GitPOAP
+
+On-chain identities are only useful if there are others to interact with and applications built to leverage them.  That’s why we decided to build on top of POAP.  
+
+GitPOAP is meant to be a platform that will enable a new era of web3 applications, and we’re excited to see what else people build on top of it.  Our public API is [now live](https://docs.gitpoap.io/api)!  Start building and feel free to reach out if there’s anything we can do to better support you.
```

Comment:
GitPOAP is a platform that will enable a new era of web3 applications, and we’re excited to see what people build on top of it. Our public API is [now live](https://docs.gitpoap.io/api)! Start building and feel free to reach out if there’s anything we can do to better support you.